html-attribute-id-addmedia (#3974)
Codecademy/docs: content/html/concepts/attributes/terms/id/id.md

</p>
```

> ![Screenshot that shows an example of a unique id attribute.](https://raw.githubusercontent.com/Codecademy/docs/main/media/html-attribute-id-example-img1.png)

Reviewer: ```suggestion
![Use of unique id attribute with inline CSS](https://raw.githubusercontent.com/Codecademy/docs/main/media/html-attribute-id-example-img1.png)
```

Clicking on the link in the examples above will direct the browser screen to the location where `id` is `"my-city"` (either on the same page or on another page).

> ![Shows how navigating a section on the same page and another page uses a unique id attribute.](https://raw.githubusercontent.com/Codecademy/docs/main/media/html-attribute-id-example2-gif.gif)

Reviewer: ```suggestion
![Use of a unique id attribute for navigation on the same and other pages in the website](https://raw.githubusercontent.com/Codecademy/docs/main/media/html-attribute-id-example2-gif.gif)
```

The HTML element with `id` is selected in CSS for styling using `#` followed by the `id` value. This applies to internal or external ruleset styling methods. The example above applies a color to the designated element, for an overview of all styling methods, please see [CSS Anatomy](https://www.codecademy.com/resources/docs/css/anatomy).

> ![CSS styling using id selector.](https://raw.githubusercontent.com/Codecademy/docs/main/media/html-attribute-id-exampl3.png)

Reviewer: ```suggestion
![CSS styling using id selector](https://raw.githubusercontent.com/Codecademy/docs/main/media/html-attribute-id-exampl3.png)
```

```

In the example above, the element is selected using the `getElementById()` method. The color of that element is then changed by assigning a new value, that is passed in as an argument of the function, to the `style.color` property.

> ![The javascript uses the id attribute to style the html element.](https://raw.githubusercontent.com/Codecademy/docs/main/media/html-attribute-id-example4.png)

Reviewer: ```suggestion
![Use of id attributes to style HTML elements in JavaScript](https://raw.githubusercontent.com/Codecademy/docs/main/media/html-attribute-id-example4.png)
```
